selfdrive/controls/lib/latcontrol_curvature.py

```diff
@@ -0,0 +1,50 @@
+import math
+
+from cereal import log
+from selfdrive.controls.lib.latcontrol import LatControl, MIN_STEER_SPEED
+from selfdrive.controls.lib.pid import PIDController
+
+STEER_ANGLE_SATURATION_THRESHOLD = 2.5  # Degrees
+
+
+class LatControlCurvature(LatControl):
+  def __init__(self, CP, CI):
+    super().__init__(CP, CI)
+    self.steer_max = 0.02
```

Comment:
this feels like a hack. I thought to use [-1.0, 1.0] and apply in carcontroller, but I think would have to test new PID coefficients?